cmd/kubeadm/app/master/discovery.go

```diff
@@ -60,7 +60,7 @@ func encodeKubeDiscoverySecretData(s *kubeadmapi.MasterConfiguration, caCert *x5
 	return data
 }
 
-func newKubeDiscoveryPodSpec() api.PodSpec {
+func newKubeDiscoveryPodSpec(s *kubeadmapi.MasterConfiguration) api.PodSpec {
```

Comment:
Weird I still don't get it in my review. Can you confirm you've committed this change @errordeveloper?